### DISCUSSION TITLE
Unable to install syslog app

### DISCUSSION THREAD

**Discussion by mortenn** [datalust/seq-tickets#2315] [Help]
I am running Seq in an Azure Container Instance but when I try installing the syslog app, it seems like seq crashes without any errors being logged. I was able to install the slack app, though

**Comment by nblumhardt** [comment]
Hi Morten, thanks for your message.

Does the container exit when installing the input from NuGet, or when starting an instance of the input once it is installed?

The most common cause of unexpected container exits is RAM exhaustion, so I'd suggest (even temporarily) trying the container with a larger memory allocation if it's currently set to something small.

Setting the Seq container's restart policy to Never (temporarily) can also help surface some logs in the event of a crash.

Let me know if either of the above helps,

Best regards,
Nick

**Comment by mortenn** [comment]
I discovered you provide a docker image for the syslog ingress, so I am using that now and it seems to work.
As I haven't set a maximum for memory and cpu, I presumed it would scale up automatically, but the terraform docs are a little terse on the matter :)

Thanks for the tip, if I go back to trying the app I will keep it in mind
